controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -1303,6 +1086,56 @@ private boolean hasConnectionFailed(Throwable unwrap) {
         return new ImmutablePair<>(rateType, desiredRate);
     }
 
+    private interface SegmentHelperLog {
```

Comment:
Nit: we dont need an additional interface here and can use `Consumer` interface for these purposes.

Reply:
thank you Shivesh, the goal is to have ()->void  instead Consumer's (T)->void - as we don't have input argument to be handled here.

Reply:
I think this is equivalent to the builtin Runnable. You can just change the type in the map and everything should work as-is without this being declared.